great_expectations/rule_based_profiler/expectation_configuration_builder/default_expectation_configuration_builder.py

```diff
@@ -7,14 +22,38 @@
 )
 from great_expectations.rule_based_profiler.types import Domain, ParameterContainer
 from great_expectations.rule_based_profiler.util import (
+    get_parameter_value,
     get_parameter_value_and_validate_return_type,
 )
 
+text = Suppress("'") + Word(alphas, alphanums) + Suppress("'")
+integer = Word(nums).setParseAction(lambda t: int(t[0]))
+var = Combine(Word("$" + alphas, alphanums + "_.") + ppOptional("[" + integer + "]"))
+comparison_operator = oneOf(">= <= != > < ==")
+bitwise_operator = oneOf("~ & |")
+operand = text | integer | var
+
+expr = infixNotation(
+    operand,
+    [
+        (comparison_operator, 2, opAssoc.LEFT),
+        (bitwise_operator, 2, opAssoc.LEFT),
+    ],
+)
```

Comment:
❤️ Brilliant and elegant (assuming covered by tests sufficiently!). :-)

Reply:
🙇🏻